waspc/src/Wasp/Generator/DockerGenerator.hs

```diff
@@ -49,7 +48,7 @@ genDockerfile spec = do
             [ "usingPrisma" .= not (null $ AS.getDecls @AS.Entity.Entity spec),
               "serverPrismaClientOutputDirEnv" .= getEnvVarDefinition serverPrismaClientOutputDirEnv,
               "dbSchemaFileFromServerDir" .= SP.fromRelFile dbSchemaFileFromServerDir,
-              "nodeMajorVersion" .= show (SV.major latestMajorNodeVersion),
+              "nodeMajorVersion" .= show (getMajorNumberOfLowerBoundOfUserNodeVersionRange spec),
```

Comment:
If the user wants their minimum node version to be `18.12.1`, this will specify only `18`. Is that ok?

Reply:
Yes, this is it! Yeah, so far we were specifying only major version and I couldn't find any reason to question that so I went with it, seemed reasonable for now. We can become more precise in the future if that will be needed.